tsconfig.base.json

```diff
@@ -27,6 +27,7 @@
       "@fluentui/react-positioning": ["packages/react-positioning/src/index.ts"],
       "@fluentui/react-shared-contexts": ["packages/react-shared-contexts/src/index.ts"],
       "@fluentui/react-menu": ["packages/react-menu/src/index.ts"],
+      "@fluentui/react-portal": ["packages/react-portal/src/index.ts"],
```

Comment:
Ah, thanks for remembering where I forgot !

Reply:
no worries, we don't have automation for this atm, blame robots! :D